### PR TITLE
Add Banner component

### PR DESCRIPTION
**Pull request by dhruska**
Adds new Banner component

image

### CODE REVIEW

File: packages/core/src/components/Banner.tsx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+import type { IconSlot } from "../interfaces/Icon";
+import shadow from "../styles/shadow";
+import { withTheme } from "../theming";
+import type { Theme } from "../styles/DefaultTheme";
+
+const ELEVATION = 1;
+const DEFAULT_MAX_WIDTH = 960;
+
+type Props = {
+  initiallyVisible: boolean;
```
Comment (smlarkin) [inline packages/core/src/components/Banner.tsx:21]: For non-optional props, can you set a default when you destructure below?

File: packages/core/src/components/Banner.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+      x: number;
+      y: number;
+      width: number;
+      height: number;
+    };
+  };
+};
+
+const Banner: React.FC<Props> = ({
+  initiallyVisible,
```
Comment (smlarkin) [inline packages/core/src/components/Banner.tsx:47]: Not passing in a value might work, but can you be explicit about non-optional props and set to false?

**Review by smlarkin — CHANGES_REQUESTED** [review]
I understand you are setting default props in the map, but wish that at the component level they were set too.  Up to you, of course.

**Review by smlarkin — DISMISSED** [review]

File: packages/core/src/components/Banner.tsx (hunk 1 above)
Comment (dhruska) [reply packages/core/src/components/Banner.tsx:21]: Definitely

File: packages/core/src/components/Banner.tsx (hunk 2 above)
Comment (dhruska) [reply packages/core/src/components/Banner.tsx:47]: Certainly, that makes sense

**Review by smlarkin — APPROVED** [review]

**Pull request merged**